src/view/configuration/configuration.jsx

```diff
@@ -518,7 +520,22 @@ const Configuration = ({ formikProps, initInfo }) => {
                         />
                       </div>
                     </div>
-
+                    <div className="u-gapTop">
+                      <InfoTipLayout tip="Prehiding snipped should be placed on the page at the very top">
+                        <FieldLabel
+                          labelFor="prehidingStyleField"
```

Comment:
`prehidingStyleField` isn't the element this is labeling, so this should change or be removed or something (I'm not sure how to properly handle accessibility for this particular situation).